client/src/components/SandwichForm.js

```diff
@@ -71,11 +71,11 @@ const SandwichForm = (props) => {
           value={sandwich.imageURL}
         />
       </label>
-      <div className="button-group">
-        <input type="submit" value="Create Sandwich"/>
-        <button onClick={clearForm} type="button">
+      <div>
+        <input className="button" type="submit" value="Create Sandwich"/>
+        <p className="button clear-button" onClick={clearForm} type="button">
           Clear
-        </button>
+        </p>
```

Comment:
Im not aware of a "type" property doing anything to a p tag. 

Unless you are trying to avoid using a button element, I would use a button. Otherwise, you can style the p tag to look like a button, but lets get rid of that "type"